src/background.ts

```diff
@@ -10,3 +10,22 @@ chrome.runtime.onInstalled.addListener((details) => {
   }
 
 })
+
+chrome.runtime.onMessage.addListener((message, sender) => {
+  if (message.type === "EXECUTE_BACKWARD" && sender.tab?.id) {
+    chrome.scripting.executeScript({
+      target: { tabId: sender.tab.id, allFrames: true },
+      world: "MAIN",
```

Comment:
This was my nightmare and why it took me 3 months from suggestion time to implementation, Chrome won't let me run injected scripted in main space